presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry.py

```diff
@@ -56,7 +56,8 @@ def load_predefined_recognizers(self):
             UsBankRecognizer(), UsLicenseRecognizer(),
             UsItinRecognizer(), UsPassportRecognizer(),
             UsPhoneRecognizer(), UsSsnRecognizer(),
-            SpacyRecognizer(), SgFinRecognizer()])
+            SpacyRecognizer(), SgFinRecognizer(),
```

Comment:
not sure i follow the logic of what happens when there is not a text analytics defined. when does the recognizer loading break? will it be a part of the recognizers list and will not function?